eahub/templates/eahub/group.html

```diff
@@ -23,18 +23,17 @@
     </div>
     <div class="col-xs-12 col-md-6">
       <h1 class="word-break">{{ group.name }}</h1>
-      {% if group.email %}
-      <a href="mailto:{{ group.email }}">
-        <div class="btn btn-default purple">
-          <i class="fa fa-envelope"></i> Message
-        </div>
-      </a>
-      {% elif group.facebook_group %}
-      <a href="{{ group.facebook_group }}" target="_blank">
-        <div class="btn btn-default purple">
-          <i class="fa fa-envelope"></i> Message
-        </div>
-      </a>
+      {% if request.user.id is not None %}
+      <div class="btn btn-default purple">
+        <a href="{% url 'message_group' group.slug  %}"><i class="fa fa-pencil-alt"></i> Send Message</a>
+      </div>
+      {% endif %}
+
+      {% if group.facebook_group %}
```

Comment:
Let's remove the facebook group "message" button, we don't really need it anymore

Reply:
done